fabric-structures-v0/src/main/java/net/fabricmc/fabric/api/structures/v0/FabricStructures.java

```diff
@@ -0,0 +1,48 @@
+package net.fabricmc.fabric.api.structures.v0;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.gen.GenerationStep;
+import net.minecraft.world.gen.feature.FeatureConfig;
+import net.minecraft.world.gen.feature.StructureFeature;
+
+/**
+ * API that hooks into the internal structure generation step logic
+ */
+public final class FabricStructures {
```

Comment:
I'd say `FabricStructureHelper` is a better name

Reply:
Resolved in a6b4507